packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -811,6 +811,7 @@ export default {
             const message = this.$t(error.messageKey) || this.$t('errorMessages.genericServerError');
             const eventToEmit = error.eventToEmit || EventNames.CheckoutFailure;
             const logMessage = error.logMessage || 'Consumer Checkout Failure';
+            const errorName = error.errorCode || '';
```

Comment:
Isn't this an `errorCode` rather than an `errorName`?

Reply:
Going by the name inside the error object, I'd say yes.. But the actual value isn't a code. The value is the name of the error, genuinely.

For a `PlaceOrderError` the value will be `PlaceOrderError` 😄

Reply:
But happy to change to remove the confusion from this line.

Reply:
Sad days 😢, haha. I'll leave it up to you 😄